src/tcgwars/logic/impl/gen3/RubySapphireNG.groovy

```diff
@@ -597,14 +700,24 @@ public enum RubySapphireNG implements LogicCardInfo {
         pokePower "Energy Trans", {
           text "As often as you like during your turn (before your attack), move a [G] Energy card attached to 1 of your Pokémon to another of your Pokémon. This power can't be used if Sceptile is affected by a Special Condition."
           actionA {
+            checkNoSPC()
+            assert my.all.findAll {it.cards.energyCount(G)>0} : "There are no Pokémon with [G] Energy cards"
+            assert my.all.size()>=2 : "There is only one Pokémon on the field"
```

Comment:
Nitpick: Couldn't you just check if there is a benched Pokémon?

Reply:
That should work fine, I think? Vaguely recall discussion on that a long time ago, most likelt when building assertMyBench() and the other similar statics.

Leaving it like this for now, can be talked further later on though, in a more general sense.